sdk/iot/Azure.Iot.Hub.Service/api/Azure.Iot.Hub.Service.netstandard2.0.cs

```diff
@@ -144,6 +144,41 @@ public partial class BulkRegistryOperationResponse
         public bool? IsSuccessful { get { throw null; } }
         public System.Collections.Generic.IReadOnlyList<Azure.Iot.Hub.Service.Models.DeviceRegistryOperationWarning> Warnings { get { throw null; } }
     }
+    public partial class CloudToDeviceMessage
+    {
+        public CloudToDeviceMessage() { }
+        public Azure.Iot.Hub.Service.Models.CloudToDeviceMessageAck? Ack { get { throw null; } set { } }
+        public string ContentEncoding { get { throw null; } set { } }
+        public string ContentType { get { throw null; } set { } }
+        public string CorrelationId { get { throw null; } set { } }
+        public System.DateTimeOffset? CreationTimeUtc { get { throw null; } set { } }
+        public string Destination { get { throw null; } set { } }
+        public System.DateTimeOffset? ExpirationTimeUtc { get { throw null; } set { } }
+        public string MessageId { get { throw null; } set { } }
+        public object MessagePayload { get { throw null; } set { } }
+        public string MessageSchema { get { throw null; } set { } }
+        public System.Collections.Generic.IDictionary<string, object> Properties { get { throw null; } set { } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct CloudToDeviceMessageAck : System.IEquatable<Azure.Iot.Hub.Service.Models.CloudToDeviceMessageAck>
+    {
+        private readonly object _dummy;
```

Comment:
What are these variables? XML comments would help.

Reply:
Autogenerated, so I can't speak to it.

We can definitely look at the explanation, once we actually begin implementing these APIs.